### PR TITLE
[NON JS version] Dynamic Lists: Subsidiaries

### PR DESCRIPTION
**Pull request by RuslanHamidullin**
[NON JS version] Dynamic Lists: Subsidiaries (Trade Form: A13)

### CODE REVIEW

File: app/controllers/form/subsidiaries_controller.rb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    result_subsidiaries.delete_if do |el|
+      el["name"] == params[:name] &&
+      el["location"] == params[:location]
+    end
+
+    result_subsidiaries = if result_subsidiaries.present?
+      result_subsidiaries
+    else
+      []
+    end
```
Comment (matthewford) [inline app/controllers/form/subsidiaries_controller.rb]: `end` at 69, 4 is not aligned with `if` at 65, 26

File: app/controllers/form/subsidiaries_controller.rb [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  expose(:remove_subsidiary_result_doc) do
+    result_subsidiaries = existing_subsidiaries
+    result_subsidiaries.delete_if do |el|
+      el["name"] == params[:name] &&
+      el["location"] == params[:location]
+    end
+
+    result_subsidiaries = if result_subsidiaries.present?
+      result_subsidiaries
+    else
```
Comment (matthewford) [inline app/controllers/form/subsidiaries_controller.rb]: Align `else` with `if`.

File: app/controllers/form/subsidiaries_controller.rb [hunk 3]
```diff
@@ -0,0 +1,120 @@
+class Form::SubsidiariesController < Form::BaseController
+
```
Comment (matthewford) [inline app/controllers/form/subsidiaries_controller.rb]: Extra empty line detected at class body beginning.

File: app/controllers/form/subsidiaries_controller.rb [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  expose(:remove_subsidiary_result_doc) do
+    result_subsidiaries = existing_subsidiaries
+    result_subsidiaries.delete_if do |el|
+      el["name"] == params[:name] &&
+      el["location"] == params[:location]
+    end
+
+    result_subsidiaries = if result_subsidiaries.present?
+      result_subsidiaries
```
Comment (matthewford) [inline app/controllers/form/subsidiaries_controller.rb]: Use 2 (not -20) spaces for indentation.

File: app/value_objects/subsidiary.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  # when JS is disabled
+
+  include ActiveModel::Model
+
+  attr_reader :name, :location, :employees
+
+  validates :name, :location, :employees, presence: true,
+                                          length: { maximum: 100 }
+
+  def initialize(attrs={})
```
Comment (matthewford) [inline app/value_objects/subsidiary.rb]: Surrounding space missing in default value assignment.

**Comment by RuslanHamidullin** [comment]
Amazing bro :bomb:

**Pull request merged**